src/Storage/Device/S3Compatible.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Utopia\Storage\Device;
+
+use function preg_replace;
```

Comment:
For consistency, let's not declare strict types and let's not import preg_replace.

Reply:
This has been resolved, sorry for the delay.